spacy/errors.py

```diff
@@ -913,6 +913,10 @@ class Errors(metaclass=ErrorsWithCodes):
     E1034 = ("Node index {i} out of bounds ({length})")
     E1035 = ("Token index {i} out of bounds ({length})")
     E1036 = ("Cannot index into NoneNode")
+    E1037 = ("Annotation is ill-formed.")
```

Comment:
Sorry, I meant we can use this error code, not this exact message. Can we provide useful information from the validation to the user in the error message?

Reply:
Passing on the message from `schema.validate()` now in case of an ill-formed object. If you feel like something else might make more sense here, feel free to reopen the conversation.